packages/flutter_test/lib/src/binding.dart

```diff
@@ -767,6 +767,11 @@ class AutomatedTestWidgetsFlutterBinding extends TestWidgetsFlutterBinding {
     });
 
     return new Future<Null>.microtask(() async {
+      // Run all queued microtasks.
+      await new Future.microtask(() {});
+      // When the test had an exception, the test-framework already
+      // ran the teardown functions, removing the _fakeAsync function.
+      if (_fakeAsync == null) return;
```

Comment:
This `return` statement and the one below are inconsistent; one returns something, the other does not.

Reply:
done.